src/parser/paladin/protection/modules/features/Checklist/Module.js

```diff
@@ -30,13 +32,15 @@ class Checklist extends BaseChecklist {
         castEfficiency={this.castEfficiency}
         extras={{
           lotpAbility: this.lotp._activeSpell,
+          hotrAbility: this.hotr._activeSpell,
```

Comment:
My understanding was that the underscore variable prefix basically meant that it shouldn't be accessed from outside the class -- is this intentional?

Reply:
you're right, i'll fix these.